src/cirq_iqm/iqm_remote.py

```diff
@@ -66,12 +66,12 @@ def run_sweep(
             params: cirq.Sweepable,
             repetitions: int = 1,
     ) -> list[cirq.Result]:
-        """Samples from the given quantum circuit.
+        """Samples from, i.e. executes, the given quantum circuit.
 
         Nontrivial sweeping is not yet supported, ``params`` should be ``None``.
 
         Args:
-            program: The circuit to sample from.
+            program: The circuit to execute.
```

Comment:
Same as above. "The circuit to sample from" comes from the parent class.

Reply:
Docstring removed here.